input/fsh/instances/questionnaire-sdc-profile-example-framingham-hchd-lhc.fsh

```diff
@@ -3,83 +3,102 @@ InstanceOf: SDCBaseQuestionnaire
 Title: "SDC-Hard Coronary Heart Disease"
 Description: "A made-up form for calculating the 10-year-risk of Hard Coronary Heart Disease. It shows how variables and calculatedExpression can be used to compute for the risk probability."
 Usage: #example
-* extension[$variable]
+* extension[0]
```

Comment:
Why change the format of declaring the extensions?

Reply:
As when using the equals one, they just blat over any others, so you just end up with 1 variable in the output!!!